src/request-pipeline/special-pages.js

```diff
@@ -0,0 +1,9 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
+export default {
```

Comment:
They are always used in pair, so it can be just an array. Also this const can be moved to url utils I guess